scripts/zkai_d128_gate_value_activation_fused_gate.py

```
@@ -0,0 +1,620 @@
+        raise FusedGateError("mutation inventory mismatch")
+    if "all_mutations_rejected" in inventory and inventory["all_mutations_rejected"] is not True:
+        raise FusedGateError("mutation inventory rejection status mismatch")
+    if "mutation_result" in payload:
+        result = require_dict(payload["mutation_result"], "mutation result")
+        if result.get("case_count") != len(MUTATION_NAMES) or result.get("all_mutations_rejected") is not True:
+            raise FusedGateError("mutation result mismatch")
+    if payload.get("payload_commitment") != payload_commitment(payload):
+        raise FusedGateError("payload commitment mismatch")
+
+
+def mutation_cases(payload: dict[str, Any]) -> list[tuple[str, Any]]:
```

Ruff (0.15.12)

[warning] Unused function argument: `payload`

(ARG001)